layouts/partials/head.html

```diff
@@ -131,19 +119,4 @@
         </script>
     {{ end }}
 
-    {{/* Tests to enforce required frontmatter for certain content types. */}}
-    {{ if and (eq .Type "blog") .IsPage }}
```

Comment:
These checks were measurably affecting the build time?

Reply:
This was actually the first thing I looked at, and no, it didn't make any measurable difference -- I just forgot to put it back in. 😄 Thanks for the catch.